### PR TITLE
Task7

### PR DESCRIPTION
**Pull request by YoriB**

### CODE REVIEW

File: src/components/Comments.jsx [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { getComments } from '../utils/api';
 const dayjs = require('dayjs')
 
 const Comments =({singleArticleID}) => {
+  console.log(singleArticleID)
```
Comment (HeatherEver) [inline src/components/Comments.jsx:7]: remove console logs please :)

File: src/utils/api.js [hunk 2]
```diff
@@ -6,38 +6,33 @@ const articlesAPI = axios.create({
...
-  const patchData = { votes : votes }
+export const patchArticleApi = (id, votes) => {  
 
-return articlesAPI.patch(`/articles/${article_id}`, patchData)
-};
-=======
->>>>>>> 934cd87762efa79f7292862d4e3f68004d414493
+
+return articlesAPI.patch(`/articles/${id}`, { inc_votes : votes }).then(data => {
+  console.log(data.data);
```
Comment (HeatherEver) [inline src/utils/api.js:35]: another console log to delete!

**Review by HeatherEver — APPROVED** [review]
Really good work Yori! Merge away

File: src/components/Comments.jsx (hunk 1 above)
Comment (YoriB) [reply src/components/Comments.jsx:7]: Ok will do that now. Thanks.:)

File: src/utils/api.js (hunk 2 above)
Comment (YoriB) [reply src/utils/api.js:35]: Will do that too. Thanks.:)

**Pull request merged**